tests/chainer_tests/functions_tests/math_tests/test_exponential_m1.py

```diff
@@ -0,0 +1,65 @@
+import unittest
+
+import numpy
+
+import chainer
+from chainer import cuda
+import chainer.functions as F
+from chainer import gradient_check
+from chainer import testing
+from chainer.testing import attr
+from chainer.testing import condition
+
+
+@testing.parameterize(*testing.product({
+    'shape': [(), (3, 2)],
+}))
+class Expm1FunctionTest(unittest.TestCase):
+
+    def setUp(self):
+        self.x = numpy.random.uniform(.5, 1, self.shape).astype(numpy.float32)
+        self.gy = numpy.random.uniform(-1, 1, self.shape).astype(numpy.float32)
+
+    def check_forward(self, op, op_np, x_data):
```

Comment:
Thank you for your comment.

I know `op` is a `Chainer.Function` and `op_np` is a corresponding NumPy function. I was thinking we do not need to feed `F.exp1` (resp. `numpy.exp1`) to `op` (resp. `op_np`) as arguments and directly assign `y = F.exp1(x)` in `check_forward` etc. That make the code simpler, although it is a matter of tastes and is not a critical problem.